modules/gdscript/gdscript_warning.h

```diff
@@ -67,7 +67,8 @@ class GDScriptWarning {
 		STANDALONE_TERNARY, // Return value of ternary expression is discarded.
 		ASSERT_ALWAYS_TRUE, // Expression for assert argument is always true.
 		ASSERT_ALWAYS_FALSE, // Expression for assert argument is always false.
-		REDUNDANT_AWAIT, // await is used but expression is synchronous (not a signal nor a coroutine).
+		REDUNDANT_AWAIT, // Await is used but expression is synchronous (not a signal nor a coroutine).
```

Comment:
Not sure why you changed this. `await` is a keyword, so it should not be capitalized.